### PR TITLE
feat: Add helm diff guide

### PR DESCRIPTION
**Pull request by alex131290**

### CODE REVIEW

File: docs/guides/DevOps/helm-diff.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+* From the extracted `tgz` file, copy the `bin` directory and the `plugin.yaml` into the `HELM_PLUGINS`/`helm-diff` directory.
+
+Now you have `helm-diff` plugin installed on your Windows workstation.
+
+## How to use it
+
+Assuming that your `chart` name is `foo` and your `values` file name is `dev-values.yaml`
+run the following from the `Chart` directory you wish to upgrade.  
+`helm diff upgrade foo --values dev-values.yaml .`  
+If there's a `diff` the output will show all the diffs between the `k8s` objects which will be modified/create/deleted.
```
Comment (netanelC) [inline docs/guides/DevOps/helm-diff.mdx]:
```suggestion
If there's a `diff`, the output will show all the diffs between the `k8s` objects which will be modified/create/deleted.
```

File: docs/guides/DevOps/helm-diff.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* Create a directory `helm-diff` under the `HELM_PLUGINS` path.
+* From the extracted `tgz` file, copy the `bin` directory and the `plugin.yaml` into the `HELM_PLUGINS`/`helm-diff` directory.
+
+Now you have `helm-diff` plugin installed on your Windows workstation.
+
+## How to use it
+
+Assuming that your `chart` name is `foo` and your `values` file name is `dev-values.yaml`
+run the following from the `Chart` directory you wish to upgrade.  
+`helm diff upgrade foo --values dev-values.yaml .`
```
Comment (netanelC) [inline docs/guides/DevOps/helm-diff.mdx:46]: Does helm-diff supports multiple values? Mention it

**Review by netanelC — CHANGES_REQUESTED** [review]
Good Job!
Add a screenshot showing an output of helm-diff

File: docs/guides/DevOps/helm-diff.mdx (hunk 2 above)
Comment (alex131290) [reply docs/guides/DevOps/helm-diff.mdx:46]: @netanelC  yes, added an example

**Comment by alex131290** [comment]
> Good Job! Add a screenshot showing an output of helm-diff

@netanelC added an image

**Review by netanelC — APPROVED** [review]

**Pull request merged**